lib/memcached/elasticache.rb

```diff
@@ -14,22 +14,25 @@ class Client
       # @param [String] config_endpoint ElastiCache config endpoint strings like "my-host.cache.aws.com:11211"
       # @option [Integer] :refresh_interval second-scale interval of refreshing cluster endpoints
       # @option [Integer] :max_retry_count max retry times to command and refresh cluster endpoints
+      # @option [Boolean] :local_mode if local_mode is true, the client always treats the config endpoint as the only cluster endpoint (for non-ElastiCache memcached)
```

Comment:
[nits]if standalone_mode is true, the client treats `config_endpoint` as standard memcached server (NO require ElastiCache endpint). It is useful during development.